### PR TITLE
[Pages] Added Zone Hold known issue

### PR DESCRIPTION
**Pull request by cf-scott**
When a domain has a Zone Hold, it is not possible to add a Custom Domain to a Pages Project.

### CODE REVIEW

File: content/pages/platform/custom-domains.md [hunk 1]
```diff
@@ -109,3 +109,11 @@ example.com.            300     IN      CAA     0 issuewild "pki.goog; cansignht
 ```
 
 Refer to the [Certification Authority Authorization (CAA) FAQ](/ssl/edge-certificates/troubleshooting/caa-records/) for more information.
+
+### Zone Holds
+
+A [Zone Hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/) will prevent Pages from adding a Custom Domain for a hostname under a Zone Hold.
+
+To add a Custom Domain for a hostname with a Zone Hold, please temporarly [release the Zone Hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) during the Custom Domain setup process.
```
Comment (Le0Developer) [inline content/pages/platform/custom-domains.md]:
```suggestion
To add a Custom Domain for a hostname with a Zone Hold, please temporarily [release the Zone Hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) during the Custom Domain setup process.
```
typo

**Comment by WalshyDev** [comment]
Can we instead add to the domain debugging section which is the place to go for anyone dealing with pending - https://developers.cloudflare.com/pages/platform/debugging-pages/#domain-stuck-in-verification

File: content/pages/platform/custom-domains.md [hunk 2]
```diff
@@ -109,3 +109,11 @@ example.com.            300     IN      CAA     0 issuewild "pki.goog; cansignht
 ```
 
 Refer to the [Certification Authority Authorization (CAA) FAQ](/ssl/edge-certificates/troubleshooting/caa-records/) for more information.
+
+### Zone Holds
+
+A [Zone Hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/) will prevent Pages from adding a Custom Domain for a hostname under a Zone Hold.
```
Comment (deadlypants1973) [inline content/pages/platform/custom-domains.md]:
```suggestion
A [Zone hold](/fundamentals/account-and-billing/account-security/zone-holds/) will prevent Pages from adding a custom domain for a hostname under a zone hold.
```

File: content/pages/platform/custom-domains.md [hunk 3]
```diff
@@ -109,3 +109,11 @@ example.com.            300     IN      CAA     0 issuewild "pki.goog; cansignht
...
+
+To add a Custom Domain for a hostname with a Zone Hold, please temporarily [release the Zone Hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) during the Custom Domain setup process.
```
Comment (deadlypants1973) [inline content/pages/platform/custom-domains.md]:
```suggestion
To add a Custom Domain for a hostname with a zone hold, temporarily [release the zone hold](/fundamentals/account-and-billing/account-security/zone-holds/#release-zone-holds) during the custom domain setup process.
```

File: content/pages/platform/custom-domains.md [hunk 4]
```diff
@@ -109,3 +109,11 @@ example.com.            300     IN      CAA     0 issuewild "pki.goog; cansignht
...
+
+Once the Custom Domain has been successfully completed, you may [reinstate the Zone Hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/#enable-zone-holds).
```
Comment (deadlypants1973) [inline content/pages/platform/custom-domains.md]:
```suggestion
Once the custom domain has been successfully completed, you may [reinstate the zone hold](https://developers.cloudflare.com/fundamentals/account-and-billing/account-security/zone-holds/#enable-zone-holds).
```

**Review by deadlypants1973 — APPROVED** [review]
Zone hold and custom domains (not to be confused with the CF feature: Custom Domains which are part of the Workers product) change for capitalization and +1 for @WalshyDev comment

File: content/pages/platform/custom-domains.md [hunk 5]
```diff
@@ -109,3 +109,11 @@ example.com.            300     IN      CAA     0 issuewild "pki.goog; cansignht
 ```
 
 Refer to the [Certification Authority Authorization (CAA) FAQ](/ssl/edge-certificates/troubleshooting/caa-records/) for more information.
+
+### Zone Holds
```
Comment (deadlypants1973) [inline content/pages/platform/custom-domains.md]:
```suggestion
### Zone holds
```

File: content/pages/platform/custom-domains.md [hunk 6]
```diff
@@ -109,3 +109,11 @@ example.com.            300     IN      CAA     0 issuewild "pki.goog; cansignht
...
+
+A [Zone hold](/fundamentals/account-and-billing/account-security/zone-holds/) will prevent Pages from adding a custom domain for a hostname under a zone hold.
```
Comment (deadlypants1973) [inline content/pages/platform/custom-domains.md]:
```suggestion
A [zone hold](/fundamentals/account-and-billing/account-security/zone-holds/) will prevent Pages from adding a custom domain for a hostname under a zone hold.
```

**Pull request closed without merging**